app.py

```diff
@@ -222,13 +230,6 @@ def topics():
         ("Not at all interested", 1),
     ]
 
-    def get_topic_preferences(account_id):  # for geting user topic preference
```

Comment:
Likewise with above -- I'm confused what this change is about and wondering if it's just a weird synchronization issue.

Reply:
My fault, I was working on old version of code.